include/aws/auth/credentials.h

```diff
@@ -111,6 +111,12 @@ struct aws_credentials_provider_profile_options {
      */
     struct aws_byte_cursor credentials_file_name_override;
 
+    /**
+     * Use a cached merged profile collection.
```

Comment:
`a cached merged profile collection`

I was really confused by this, until I read through the code. So, maybe worth to clarify that the merged means it has both config-file-based profile collection and a credentials-file-based profile collection?

Reply:
Thanks, fixed.